privacyidea/api/token.py

```diff
@@ -1225,3 +1226,58 @@ def delete_tokeninfo_api(serial, key):
     success = count > 0
     g.audit_object.log({"success": success})
     return send_result(success)
+
+
+@token_blueprint.route('/group/<serial>/<groupname>', methods=['POST'])
+@token_blueprint.route('/group/<serial>', methods=['POST'])
+@admin_required
+@prepolicy(check_base_action, request, ACTION.TOKENGROUPS)
+@event("token_assign_group", request, g)
+@log_with(log)
+def assign_tokengroup_api(serial, groupname=None):
+    """
+    Assigns a token to a given tokengroup.
+
+    If no groupname is given, we expect a body data "groups", that
+    contains a list of tokengroups. tokengroups that are
+    not contained in this list, will be removed.
+
+    :jsonparam basestring serial: the serial number of the token
+    :jsonparam basestring groupname: The name of the tokengroup
```

Comment:
please describe parameter "groups".
Which is used in line 1255.

Reply:
done in cfc6b42a